tools/install_charm_ubuntu.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Conversion to Makefile will be completed at a later stage when we are planning for the big releases.